Insights/InsightsTests/InsightsTests.swift

```diff
@@ -2,35 +2,89 @@
 //  InsightsTests.swift
 //  InsightsTests
 //
-//  Created by Robert Mogos on 04/05/2018.
 //  Copyright © 2018 Algolia. All rights reserved.
 //
 
 import XCTest
 @testable import Insights
 
 class InsightsTests: XCTestCase {
-    
-    override func setUp() {
-        super.setUp()
-        // Put setup code here. This method is called before the invocation of each test method in the class.
+  
+  override func setUp() {
```

Comment:
nit: can remove setup and teardown since we re not doing anything with them

Reply:
done